### DISCUSSION TITLE
Debug Service not active

### DISCUSSION THREAD

**Discussion by devinhh** [codefori/vscode-ibmi#1198] [Q&A] [answered]
I am receiving this message when trying to debug a program:

EQAVS1007E <myibmihost> on port 8005 could not be connected.
Message received: connect ECONNREFUSED nnn.nnn.nnn.nnn:8005

**Comment by worksofliam** [comment, marked as answer]
Have you checked out our Debugging docs? It notes about starting the debug service.

https://halcyon-tech.github.io/docs/#/pages/developing/debug/debug

**Reply by devinhh** [reply]
That was exactly what I needed .  Thanks
